test/callback.tests.js

```diff
@@ -960,20 +960,13 @@ describe('callback response_mode: form_post', () => {
           scope: 'openid profile email',
         },
         afterCallback: async (req, res, session) => {
-          const userInfo = await req.oidc.fetchUserInfo();
```

Comment:
This call is not possible as it requires `accessToken`, which is not present in the request context yet. As I understand this is not critical part of the test, important is what this calback returns and if it is put into `authSession`. So I added that data directly here.